src/solidity-helpers.ts

```diff
@@ -12,15 +13,15 @@ export function soliditySha3(...values: any[]): string {
  */
 export class Web3Signer {
```

Comment:
You should create a new `OfflineWeb3Signer` class and leave the current `Web3Signer` unchanged.